roles/config-rh-sso/tasks/manage-clients.yml

```diff
@@ -75,5 +75,51 @@
     protocol_mappers: "{{ client.protocol_mappers | default(omit) }}"
     attributes: "{{ client.attributes | default(omit) }}"
     public_client: "{{ client.public_access | default(false) }}"
+  register: rh_sso_client
   when:
     - invalid_field is undefined or invalid_field|trim == ""
+
+- name: "Get Authentication Flows For Realm: {{ client.realm }}"
+  uri:
+    url: "{{ rh_sso_protocol }}://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/admin/realms/{{ client.realm }}/authentication/flows"
+    method: GET
+    headers:
+      Authorization: "Bearer {{ rh_sso_token.json.access_token }}"
+  register: client_flows
```

Comment:
We need the `validate_certs` variable defined here too.

Reply:
Done